docs/concept-workflows.md

```diff
@@ -1,30 +1,210 @@
 ---
 id: concept-workflows
-title: Workflows
+title: The concept of Workflows
 sidebar_label: Workflows
 description: Temporal core abstraction is a fault-oblivious stateful Workflow. The state of the Workflow code, including local variables and threads it creates, is immune to process and Temporal service failures.
 ---
 
 import { ResponsivePlayer } from '../src/components'
 
-## Overview
+## Focus on business logic
 
-Temporal core abstraction is a **fault-oblivious stateful Workflow**. The state of the Workflow code, including local variables and threads it creates, is immune to process and Temporal service failures.
-This is a very powerful concept as it encapsulates state, processing threads, durable timers and event handlers.
+Depending on the language, a Workflow is really just a single function or object method that orchestrates a series of actions.
```

Comment:
I fear that people that read this would interpret "just a single function or object method that orchestrates a series of actions" as the whole workflow has to be written in a single function. We need to emphasize (not necessary here) that workflows have the full power of abstraction (OO, functions, etc) of the host language.